### PR TITLE
setup redux

### PR DESCRIPTION
**Pull request by Dan-Y-Ko**
Setup the redux store and used the response coming from the backend as the model for the soup kitchen location state.

I implemented just a basic integration of the store using useSelector in the MapList component which gets passed down as props to LocationList component and renders the address in text. I have verified it works and covered loading and error states along with success state.

Also updated documentation about the ip address because it was moved and I also changed it a bit, but it's perfectly doable with the original method as described in the updated documentation.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -64,12 +64,25 @@ Then you should be able to run `expo start` to access a local build of the app o
 
 Note: To test the app with dummy data, you will have to run a local instance of the backend in another terminal window. Backend repo/instructions: https://github.com/HotSoupRepos/HotSoup-backend
 
-In the MapList screen component, there is a hardcoded IP address, you will have to change it to your own computer's IPv4 address to make requests to the running backend. You CANNOT use localhost, since the app is running on your phone/simulator.
+You can find the api request url at src/service/soupKitchenServce.ts. There is a hardcoded url including a unique IP address stored in a variable. You will have to change it to your own computer's IPv4 address to make requests to the running backend. You CANNOT use localhost, since the app is running on your phone/simulator.
```
Comment (lindentree) [inline README.md:67]: Typo here in the spelling of service I think?

Comment (Dan-Y-Ko) [reply README.md:67]: woops! will fix

**Comment by Dan-Y-Ko** [comment]
Just realized I exported the slice and I don't really need to do that (since I'm exporting the reducer from the slice specifically), so I removed that and pushed it up.

File: README.md (hunk 1 above)
Comment (rweitzman) [reply README.md]: is this resolved? if so, let's mark the conversation as resolved

**Review by rweitzman — APPROVED** [review]

**Comment by rweitzman** [comment]
Please wait to merge until we have merged dev into main!!!

**Pull request merged**